.github/workflows/ci.yml

```diff
@@ -111,8 +111,8 @@ jobs:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
 
   core-test:
-    needs: [prep-testbed, commit-lint, lint-flake-8]
-    runs-on: ubuntu-20.04
+    needs: [prep-testbed, commit-lint, lint-flake-8, check-docstring]
+    runs-on: ubuntu-18.04
```

Comment:
Max gave the reasoning for `20.04` at #2011. Is there any good reason to switch it back?